lib/lightning/workorders/export_audit.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Lightning.WorkOrders.ExportAudit do
+  @moduledoc """
+  Module to log history export actions as audit events.
+  """
+  use Lightning.Auditing.Audit,
+    repo: Lightning.Repo,
+    item: "history_export",
+    events: ["started"]
```

Comment:
@elias-ba As a matter of interest, are you planning on adding an event for completion of the export?

Reply:
I initially was tracking the enqueued, started, and completed events but then @stuartc mentioned that what we are really looking at here is being able to know who initiated a history export. Not tracking the workflow of an export. Stu any idea on this ?

Reply:
@elias-ba Ah, , no worries, was just curious.